### ISSUE TITLE
Ability to choose the branch to use

### ISSUE THREAD

**Issue by RobertAudi** [fnichol/chef-homesick#3]
It would be cool if I could choose the branch to use from the git repo, via an additional attribute called branch maybe.

**Comment by fnichol** [comment]
I don't think that feature is supported upstream in the [homesick gem](https://github.com/technicalpickles/homesick). However, if such an option was added I'd love to see this!

**Comment by RobertAudi** [comment]
Isn't that just a matter of doing a `git checkout <branch>` from the castle before doing a symlink?
